src/helper/stream/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { Context } from '../../context'
+import { StreamingApi } from '../../utils/stream'
+
+interface SSEMessage {
+  data: string
+  event?: string
+  id?: string
+}
+
+class SSEStreamingApi extends StreamingApi {
+  constructor(originalStream: StreamingApi) {
+    super(originalStream['writable']) // Accessing the private writable property - normally not recommended but works for this specific solution.
+  }
+
+  async writeSse(message: SSEMessage) {
+    const sseData =
+      [
+        message.event && `event: ${message.event}`,
+        message.id && `id: ${message.id}`,
+        `data: ${message.data}`,
+      ]
+        .filter(Boolean)
+        .join('\n') + '\n'
+
+    await this.write(sseData)
+  }
+}
+
+function setSSEHeaders(context: Context) {
```

Comment:
Use the following syntax to be uniform with others.

```ts
const setSSEHeaders = () => {}
```

Reply:
fixed!